docs/index.rst

```diff
@@ -31,6 +31,7 @@ separately from the rest of Baseplate.
    :titlesonly:
 
    baseplate: General purpose helpers <baseplate/index>
+   baseplate.chronos: Extensions to the standard library's datetime module <baseplate/chronos>
```

Comment:
how about `datetime` as the module name?

Reply:
I named this module `chronos` so that it wouldn't clash with the OG `datetime` module when importing both or confuse users... if that's not a serious concern, I can definitely rename it to `datetime` 😎

Reply:
yeah, that should be fine. we've already got `baseplate.random` for stuff that adds onto the `random` module in the stdlib.